bigtable/client/rpc_backoff_policy.cc

```diff
@@ -46,13 +46,17 @@ void ExponentialBackoffPolicy::setup(grpc::ClientContext& /*unused*/) const {}
 std::chrono::milliseconds ExponentialBackoffPolicy::on_completion(
     grpc::Status const& status) {
   using namespace std::chrono;
-  // TODO(#57) - we also need to randomize the sleep period.
-  auto delay = duration_cast<milliseconds>(current_delay_);
-  current_delay_ *= 2;
-  if (current_delay_ >= maximum_delay_) {
-    current_delay_ = maximum_delay_;
+  std::uniform_int_distribution<int> rng_distribution(
+      duration_cast<microseconds>(current_delay_range_).count() / 2,
```

Comment:
The `duration_cast<microseconds>` is redundant here, but harmless, leave it if you want, or remove it before merging.